engines/macventure/detection.cpp

```diff
@@ -86,7 +86,7 @@ class MacVentureMetaEngineDetection : public AdvancedMetaEngineDetection {
 public:
 	MacVentureMetaEngineDetection() : AdvancedMetaEngineDetection(MacVenture::gameDescriptions, sizeof(ADGameDescription), macventureGames) {
 		_guiOptions = GUIO1(GUIO_NOMIDI);
-		_md5Bytes = 5000000; // TODO: Upper limit, adjust it once all games are added
+		_md5Bytes = 5000;
```

Comment:
This will affect all the checksums, because now it takes only the first 5000 bytes. All the entries have bigger sizes than that, thus, the md5s will be different.

Also, please, just drop this line completely, as 5000 is set in the AD constructor.